setup.py

```diff
@@ -33,16 +33,16 @@
     include_package_data=True,
     zip_safe=False,
     extras_require=dict(
-        test=["plone.app.testing", "zope.testing", "plone.app.robotframework"],
+        test=["plone.app.testing", "zope.testing"],
     ),
     install_requires=[
-        "setuptools",
-        "six",
-        "zope.component",
-        "zope.interface",
-        "zope.i18nmessageid",
+        "plone.base",
         "Products.CMFCore",
         "Products.CMFPlone",
         "Products.GenericSetup >= 2.0",
+        "setuptools",
+        "zope.component",
+        "zope.i18nmessageid",
+        "zope.interface",
```

Comment:
if plone.base is used, the idea is to not depend any longer on the stack below, like CMFCore,  GenericSetup, zope.component, zope.i18nmessageid, zope.interface here.

Reply:
if the package still depends directly on `zope.interface` why should that be removed? give a go on `z3c.dependencychecker` output: https://jenkins.plone.org/view/Dependencies/job/qa-undeclared-dependencies/lastSuccessfulBuild/artifact/deps.txt/*view*/

IMHO we should add a GHA/jenkins job to check them on every package, as doing clean ups now and then is just adding noise on the commit history.

Hopefully I make it to the Alpine Sprint (it is still not 100% sure 🍀 🤞🏾 ) and I have the time to implement this 😄

Reply:
Done.
But yes, it would be nice to use z3c.dependencychecker.